Headers/DebugServer2/Target/Darwin/MachOProcess.h

```diff
@@ -21,6 +22,7 @@ class MachOProcess : public ds2::Target::POSIX::Process {
 protected:
   std::string _auxiliaryVector;
   Address _sharedLibraryInfoAddress;
+  Host::Darwin::Mach _mach;
```

Comment:
I think this (and the getter that goes with it on line 40) should be in DarwinProcess instead. The Mach interface is not associated with the binary format, but rather the OS you're running on.

Also, I might get rid of `ELFProcess` and `MachOProcess` at some point soon, as they both seem like questionable design, and we can integrate their functionality in a mixin I think.